src/server/rpc/procedures/iex-trading/iex-trading.js

```diff
@@ -0,0 +1,130 @@
+// Data attributed to https://iextrading.com/api-exhibit-a
+
+const ApiConsumer = require('../utils/api-consumer');
+
+
+let StockConsumer = new ApiConsumer('iex-trading', 'https://api.iextrading.com/1.0',{cache: {ttl: 3600*24*30*6}});
+
+const queryOpts = {
+    queryString: '/stock/',
+    headers:{
+    },
+    body: undefined,
+    cache: true
+};
+
+let stockParser = resp => {
```

Comment:
As `company` is not defined, I would remove this function and replace its use on line 94 with a similar approach used in the other functions.